pytorch_blade/bazel_build.py

```diff
@@ -171,6 +171,14 @@ def __init__(self, *args, **kwargs):
         if is_enable_quantization:
             self.torch_extra_opts.append("--config=torch_enable_quantization")
 
+        # ----------------------------------------------------------------------- #
+        # --------------------   Settings for Neural Engine   ------------------- #
+        # ----------------------------------------------------------------------- #
+        is_enable_neural_engine = os.getenv("TORCH_BLADE_ENABLE_NEURAL_ENGINE", None)
```

Comment:
env vars are usually read in `torch_blade_build.py`, and it's better to follow `ON`/`OFF` convention.

Reply:
done